test/e2e/kubectl.go

```diff
@@ -238,6 +240,39 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl run", func() {
+		It("should create an rc from an image", func() {
+			nsFlag := fmt.Sprintf("--namespace=%v", ns)
+			image := "nginx"
+			rcName := "e2e-test-nginx-rc"
+
+			By("running the image " + image)
+			runKubectl("run", rcName, "--image="+image, nsFlag)
+			By("verifying the rc " + rcName + " was created")
+			rc, err := c.ReplicationControllers(ns).Get(rcName)
+			if err != nil {
+				Failf("Failed getting rc %s: %v", rcName, err)
+			}
+			containers := rc.Spec.Template.Spec.Containers
+			if containers == nil || len(containers) != 1 || containers[0].Image != image {
+				Failf("Failed creating rc %s for 1 pod with expected image %s", rcName, image)
+			}
+
+			By("verifying the pod controlled by rc " + rcName + " was created")
+			label := labels.SelectorFromSet(labels.Set(map[string]string{"run": rcName}))
+			podlist, err := c.Pods(ns).List(label, fields.Everything())
+			if err != nil {
+				Failf("Failed getting pod controlled by rc %s: %v", rcName, err)
+			}
+			pods := podlist.Items
+			if pods == nil || len(pods) != 1 || len(pods[0].Spec.Containers) != 1 || pods[0].Spec.Containers[0].Image != image {
+				Failf("Failed creating 1 pod with expected image %s", rcName, image)
+			}
+
+			runKubectl("stop", "rc", rcName, nsFlag)
```

Comment:
Put this in an `AfterEach(func(){ ... })` so that we always cleanup rc even if test fails.